commons/active/opensmile/open_smile_2_audio_analysis.avsc

```diff
@@ -6,6 +6,8 @@
   "fields": [
     {"name": "time", "type": "double", "doc": "Device timestamp in UTC (s)."},
     { "name": "timeCompleted", "type": "double", "doc": "Timestamp in UTC (s) when subject completes the audio questionnaire." },
-    {"name": "data", "type": ["null", "string"], "doc": "Audio features computed by openSMILE recorded in Base64 format.", "default": null}
+    {"name": "config", "type": ["null", "string"], "doc": "OpenSMILE configuration file.", "default": null},
+    {"name": "format", "type": ["null", "string"], "doc": "Format of the recorded data.", "default": null},
```

Comment:
Is this audio format? If this is an analysis, will it have a format? What available formats are there? E.g. link to Android audio output file formats, or state that this means for example csv file.